wdl/ExpansionHunter.wdl

```diff
@@ -42,24 +44,43 @@ workflow ExpansionHunter {
         reference_fasta_index,
         reference_fasta + ".fai"])
 
-    call RunExpansionHunter {
+    call SplitVariantCatalog {
```

Comment:
Since this is the same for every sample, definitely want to move this to the `Scatter` wdl and run outside of the scatter and pass in the scattered catalog to this wdl.

Reply:
That will lead to splitting variant catalog only once, compared to the current implementation that splits for every sample. It is a very good improvement, thanks for the suggestion.